packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/StocktakeLineEdit.tsx

```diff
@@ -8,6 +8,8 @@ import {
   Box,
   ModalMode,
   useNotification,
+  TableProvider,
```

Comment:
This is updating the StocktakeLineEdit modal to be disabled when finalised. 

Adding a `TableProvider` around the modal / tables and then just passing `isDisabled` into the `DataTable`